.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,41 @@
+name: "CodeQL Analysis - Backend"
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - main
+      - development
+  schedule:
+    - cron: '0 7 * * 1' # Runs every Sunday at 2 am
```

```suggestion
    - cron: '0 7 * * 1' # Runs every Monday at 7 am
```
Missing comment correction.

---

Done